contracts/factory/src/contract.rs

```diff
@@ -201,12 +201,18 @@ impl FactoryTrait for Factory {
             token_wasm_hash,
             lp_init_info.clone(),
             factory_addr,
-            config.lp_token_decimals,
             share_token_name,
             share_token_symbol,
         )
             .into_val(&env);
 
+        //if let PoolType::Xyk = pool_type {
+        //    init_fn_args.push_back(config.lp_token_decimals.into_val(&env));
+        //}
+
+        //init_fn_args.push_back(share_token_name.into_val(&env));
+        //init_fn_args.push_back(share_token_symbol.into_val(&env));
```

Comment:
How was that working? Those parameters are used here: https://github.com/Phoenix-Protocol-Group/phoenix-contracts/blob/main/contracts/pool/src/contract.rs#L48-L50